README.md

```diff
@@ -0,0 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Budget App
+> Budget App: Show spending and transactions history. Users can create a category, an add transactions to it.
+
+## Built With
+- Rails
+- CSS
+- PostgreSQL
+
+## Online Version Link
+This project is available online through the following link.
+https://budget-app.nduatikagiri.co.ke
```

Comment:
- [ ] Kindly make sure that your project is  deployed and accessible online, currently this link is not accessible, please fix it as it is part of the [technical requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#technical-requirements) 👍🏿